File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/member/ChangebankLogLevel.java
package com.example.cf_sdk.logic.changebanksdk.model.member;

import com.google.gson.annotations.SerializedName;

/**
 * Constants for log type
 */

public enum ChangebankLogLevel {
    @SerializedName("DEBUG")
    DEBUG("Debug"),

    @SerializedName("ERROR")
    ERROR("Error");

    String mLevelName;

    ChangebankLogLevel(String levelName) {
        mLevelName = levelName;
    }

    public String getLevelName() {
        return mLevelName;
    }
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/member/ChangebankLogLevel.kt
package com.example.cf_sdk.changebankapi.model.member

import com.google.gson.annotations.SerializedName

/**
 * Constants for log type
 */
enum class ChangebankLogLevel(levelName: String) {
    @SerializedName("DEBUG")
    DEBUG("Debug"), @SerializedName("ERROR")
    ERROR("Error");

    var levelName = ""

    init {
        this.levelName = levelName
    }
}